experimental/web/index.html

```diff
@@ -150,9 +170,8 @@
                       ) +
                       ') -> ' +
                       status +
-                      '\n  label: [' +
-                      label +
-                      ']';
+                      '\n  label: ' +
+                      JSON.stringify(label);
```

Comment:
It seems safer to convert the message instance to a normal object via `.toObject` before stringifying it, to make sure we don't stringify fields related to the functioning of protobuf.js.

Reply:
It seems fine to just stringify those objects, they even do it in their own doc: https://www.npmjs.com/package/protobufjs#using-the-js-api
In practice I don't seem to get other fields in the stringified representation anyways.

Reply:
All good then :)